webapp/views.py

```diff
@@ -178,22 +183,29 @@ def advantage_view():
                     contract["entitlements"] = entitlements
                     contract["contractInfo"][
                         "createdAtFormatted"
-                    ] = datetime.datetime.strptime(
+                    ] = datetime.strptime(
                         contract["contractInfo"]["createdAt"],
                         "%Y-%m-%dT%H:%M:%S.%fZ",
                     ).strftime(
                         "%d %B %Y"
                     )
-                    if "effectiveFrom" in contract["contractInfo"]:
-                        contract["contractInfo"][
-                            "effectiveFromFormatted"
-                        ] = datetime.datetime.strptime(
-                            contract["contractInfo"]["effectiveFrom"],
-                            "%Y-%m-%dT%H:%M:%S.%fZ",
-                        ).strftime(
-                            "%d %B %Y"
+
+                    contract["contractInfo"]["status"] = "active"
+                    if contract["contractInfo"].get("effectiveTo"):
+                        effectiveTo = datetime.strptime(
+                            contract["contractInfo"]["effectiveTo"],
+                            "%Y-%m-%dT%H:%M:%SZ",
```

Comment:
Thought you were gonna use dateutil?

![image](https://user-images.githubusercontent.com/519935/78081783-4c00a780-73a9-11ea-85b5-5a43be97f194.png)